app/pathogen/arbovirus/wip/sections.tsx

```diff
@@ -34,9 +35,9 @@ export const redesignedArbovirusPageSections = [{
     <section
       key={input.key}
       ref={input.ref}
-      className="w-full h-[95%] bg-yellow-500 snap-start snap-always scroll-smooth"
+      className="w-full h-[95%] snap-start snap-always scroll-smooth overflow-scroll"
```

Comment:
I'm aware that having `overflow-y-scroll` inside an eleement in a scroll section causes a bit of problems with the scroll. I've got a plan but I want to move everything over first and then I'll show you the fixed scroll.

Reply:
Here's the PR, it makes the scrolling a lot better https://github.com/PathoTracker/Pathotracker/pull/151